source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -353,9 +353,10 @@ has the following properties:
 
       /**
        *  Mark all current Connections as stale, clear the WaitQueue, and mark the pool as "paused".
+       *  closeInUseConnections specifies whether a pool SHOULD force closing "in use" connections.
        *  No connections may be checked out or created in this pool until ready() is called again.
        */
-      clear(): void;
+      clear(closeInUseConnections: Boolean): void;
```

Comment:
Optional with default false?

Reply:
I think `closeInUseConnections` can be optional, especially for drivers without background thread.
@patrickfreed what do you think?